docs/api/baseplate/observers/statsd.rst

```diff
@@ -30,6 +30,10 @@ the StatsD metrics observer.
    # if not specified, metrics will only be emitted to debug logs.
    metrics.endpoint = statsd.local:8125
 
+   # optional: the fraction of statsd metrics to sample
+   # if not specified, it will default to 1.0 (all metrics sent)
```

Comment:
would be good to add a note that they must be passing config to the `Baseplate` constructor for this to work